static/app/components/group/replayCount.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled';
+
+import Placeholder from 'sentry/components/placeholder';
+import {IconPlay} from 'sentry/icons';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {useLocation} from 'sentry/utils/useLocation';
+
+type Props = {
+  groupId: string;
+  orgId: string;
+};
+
+function ReplayCount({orgId, groupId}: Props) {
+  const location = useLocation();
+
+  const eventView = EventView.fromNewQueryWithLocation(
+    {
+      id: '',
+      name: `Errors within replay`,
+      version: 2,
+      fields: ['replayId', 'count()'],
+      query: `issue.id:${groupId} !replayId:""`,
+      projects: [],
+    },
+    location
+  );
+
+  return (
+    <DiscoverQuery eventView={eventView} orgSlug={orgId} location={location} useEvents>
+      {({isLoading, tableData}) => {
+        if (isLoading) {
+          return <Placeholder width="24px" height="14px" />;
+        }
+
+        const replayCount = tableData?.data?.length ?? 0;
+        if (replayCount > 0) {
+          return (
+            <ReplayCountContainer data-test-id="replay-count">
```

Comment:
can we make this clickable, it can point to the page at `/organizations/${org}/issues/${groupid}/replays/`

also, lets wrap it with a tooltip so people know what it means.